src/applications/travel-pay/routes.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
-import { Router, Route } from 'react-router';
-import TravelPayStatusApp from './containers/TravelPayStatusApp.jsx';
+import { Switch, Route } from 'react-router-dom';
+import TravelPayStatusApp from './containers/TravelPayStatusApp';
+import TravelClaimDetails from './components/TravelClaimDetails';
 
 const routes = (
-  <Router>
-    <Route path="/" component={TravelPayStatusApp} />
-  </Router>
+  <Switch>
+    <Route exact path="/" title="TravelPayHome">
+      <TravelPayStatusApp />
+    </Route>
+    <Route exact path="/:id">
+      <TravelClaimDetails />
+    </Route>
```

Comment:
If we can, let's put this route behind the details feature flag.

Reply:
Good catch. On the off chance someone tries to navigate directly to a details route, I think I'll initially throw that check in `TravelClaimDetails.jsx`. That way they can still get redirected to the status page if they have access to that but not details.